airflow/providers/jenkins/hooks/jenkins.py

```diff
@@ -46,3 +47,24 @@ def __init__(self, conn_id: str = default_conn_name) -> None:
     def get_jenkins_server(self) -> jenkins.Jenkins:
         """Get jenkins server"""
         return self.jenkins_server
+
+    def get_build_building_state(self, job_name: str, build_number: int) -> bool:
+        """Get build building state"""
+        try:
+            if not build_number:
+                self.log.info(f"Build number not specified, getting latest build info from Jenkins")
+                job_info = self.jenkins_server.get_job_info(self.job_name)
+                build_number_to_check = job_info['lastBuild']['number']
+            else:
+                build_number_to_check = self.build_number
```

Comment:
There is no `self.build_number` in the hook. I guess this is a mistake?

Reply:
my bad, fixed